migration/tests.py

```diff
@@ -93,6 +177,22 @@ def setUp(self):
         self.source_tool_id: int = int(os.getenv('SOURCE_TOOL_ID', '0'))
         self.target_tool_id: int = int(os.getenv('TARGET_TOOL_ID', '0'))
 
+        self.test_external_tool_tab = ExternalToolTab(
+            id='context_external_tool_99999',
+            label='Test External Tool',
+            tool_id=99999,
+            is_hidden=True,
+            position=30
+        )
+
+    def test_find_tab_by_tool_id_returns_tab(self):
+        tab = CourseManager.find_tab_by_tool_id(99999, [self.test_external_tool_tab])
+        self.assertTrue(isinstance(tab, ExternalToolTab))
+
+    def test_find_tab_by_tool_id_returns_none(self):
+        tab = CourseManager.find_tab_by_tool_id(100000, [self.test_external_tool_tab])
+        self.assertTrue(tab is None)
```

Comment:
I'm just moving these from `AccountManagerTestCase`, they were in the wrong place.